dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_postgresql.sql

```diff
@@ -588,6 +588,26 @@ create index user_id_index on t_ds_project (user_id);
 CREATE UNIQUE INDEX unique_name on t_ds_project (name);
 CREATE UNIQUE INDEX unique_code on t_ds_project (code);
 
+--
+-- Table structure for table t_ds_project_parameter
+--
+
+DROP TABLE IF EXISTS t_ds_project_parameter;
+CREATE TABLE t_ds_project_parameter (
+  id int NOT NULL  ,
+  param_name varchar(255) NOT NULL ,
+  param_value varchar(255) NOT NULL ,
+  code bigint NOT NULL,
+  project_code bigint NOT NULL,
+  user_id int DEFAULT NULL ,
+  create_time timestamp DEFAULT CURRENT_TIMESTAMP ,
+  update_time timestamp DEFAULT CURRENT_TIMESTAMP ,
+  PRIMARY KEY (id)
+) ;
+
+CREATE UNIQUE INDEX unique_project_parameter_name on t_ds_project_parameter (param_name);
```

Comment:
The parameter name in a project should be unique. But it should be allowed to exist in another project. So this here need to be "CREATE UNIQUE INDEX unique_project_parameter_name on t_ds_project_parameter (project_code,param_name);"

Reply:
Sure